corehq/messaging/scheduling/templates/scheduling/create_schedule.html

```diff
@@ -5,8 +5,7 @@
 
 
 {% block js %}{{ block.super }}
-    <script src="{% static 'locations/js/widgets_main.js' %}"></script>
-    <script src="{% static 'data_interfaces/js/make_read_only.js' %}"></script>
+    {% requirejs_main 'locations/js/widgets_main' %}
```

Comment:
Similar to the above comment, this page should have its own `create_schedule_main` module, rather than adding `scheduling` and `data_interfaces` dependencies to `locations/js/widgets_main`.